lib/em-http/http_connection.rb

```diff
@@ -84,14 +151,12 @@ def host
         parent.connopts.host
       end
 
-      def certificate_store
-        @certificate_store ||= begin
```

Comment:
What does this line exactly do? 
```
@certificate_store ||= begin
```
I was curious as to why it's removed

Reply:
In Ruby, a variable whose name begins with `@` is an _instance variable_ (what other languages might call a "class member" or something).

The `||=` syntax roughly means "assign if the variable’s existing value is not `nil`".

So, the existing code stored the certificate store in an instance variable so that it could be mutated on each invocation of `ssl_verify_peer`. Since we don't need to do this any more, I removed the instance variable.